src/input/importOperations.ts

```diff
@@ -383,7 +383,8 @@ const importFromJSONTypeC = (contents: string): Operation[] => {
  *          Imported transactions
  */
 const importOperations = (path: string): Operation[] => {
-  const contents = getFileContents(path);
+  const contents = getFileContents(path)!;
```

Comment:
a bit cryptic / unreadable
would be better to ensure that `getFileContents` is throwing when not being able to get content

Reply:
This is what it does, but I do have compilation errors now if I do not allow this function to return `void`.

Reply:
Removed.